proposals/exception-handling/Exceptions-formal-examples.md

```diff
@@ -153,11 +153,11 @@ Use the trivial throw context `T` again, this time to match the throw to the `ca
 Location of a rethrown exception. Let `x, y, z` be tag indices of tags with type `[t_x]→[]`, `[t_y]→[]`, and `[t_z]→[]` respectively. Let `val_p : t_p` for every `p` among `x, y, z`.
 
 ```
-try
+try $label2
```

Comment:
I think`$label2` should be here, given that it points to this `try`'s corresponding `catch`.. No?